src/pages/deposit/Deposit.tsx

```diff
@@ -40,13 +40,163 @@ function Deposit({ poolName }: Props): ReactElement | null {
   const [shouldDepositWrapped, setShouldDepositWrapped] = useState(POOL.name === USDC_AM3D_POOL_NAME ? true : false)
+
+  const fetchFraxBalanceOnETH = async () => {
+    try {
+      if (fraxContractOnETH && account) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        const hasFraxBalance = (await fraxContractOnETH.balanceOf(account)).gt(Zero)
+        if (hasFraxBalance) {
+          setHasFrax(true);
+        }
+      }
+    } catch (err) {
+      console.error('[fetchFraxBalanceOnETH] ==> err', err)
+    }
+  }
+
+  const fetchFraxBalanceOnBSC = async () => {
+    try {
+      if (fraxContractOnBSC && account) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        const hasFraxBalance = (await fraxContractOnBSC.balanceOf(account)).gt(Zero)
+        if (hasFraxBalance) {
+          setHasFrax(true)
+        }
+      }
+    } catch (err) {
+      console.error('[fetchFraxBalanceOnBSC] ==> err', err)
+    }
+  }
+
+  const fetchFraxBalanceOnMATIC = async () => {
+    try {
+      if (fraxContractOnMATIC && account) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        const hasFraxBalance = (await fraxContractOnMATIC.balanceOf(account)).gt(Zero)
+        if (hasFraxBalance) {
+          setHasFrax(true)
+        }
+      }
+    } catch (err) {
+      console.error('[fetchFraxBalanceOnMATIC] ==> err', err)
+    }
+  }
+
+  const fetchFraxBalanceOnARBITRUM = async () => {
+    try {
+      if (fraxContractOnARBITRUM && account) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        const hasFraxBalance = (await fraxContractOnARBITRUM.balanceOf(account)).gt(Zero)
+        if (hasFraxBalance) {
+          setHasFrax(true)
+        }
+      }
+    } catch (err) {
+      console.error('[fetchFraxBalanceOnARBITRUM] ==> err', err)
+    }
+  }
+
+  const fetchFraxBalanceOnFANTOM = async () => {
+    try {
+      if (fraxContractOnFANTOM && account) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        const hasFraxBalance = (await fraxContractOnFANTOM.balanceOf(account)).gt(Zero)
+        if (hasFraxBalance) {
+          setHasFrax(true)
+        }
+      }
+    } catch (err) {
+      console.error('[fetchFraxBalanceOnFANTOM] ==> err', err)
+    }
+  }
+
+  const fetchFraxBalanceOnMOONRIVER = async () => {
+    try {
+      if (fraxContractOnMOONRIVER && account) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+        const hasFraxBalance = (await fraxContractOnMOONRIVER.balanceOf(account)).gt(Zero)
+        if (hasFraxBalance) {
+          setHasFrax(true)
+        }
+      }
+    } catch (err) {
+      console.error('[fraxContractOnMOONRIVER] ==> err', err)
+    }
+  }
+
+  const fetchFraxBalanceOnBOBA = async () => {
+    try {
+      if (fraxContractOnBOBA && account) {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
Lets not abuse the `disabling` of `lint` linter is here to help us write better code not to fight with it.